custom/gsoc/note/api/build.gradle

```diff
@@ -0,0 +1,26 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+description = 'GSoC Type-Safe API project: Note Api'
```

Comment:
... maybe it would be good to come up with a different endpoint name in the first place... something like "/ng/..." ("new generation") or "/v3/..." .... have to see what will be good to keep permanently (once this is done it will stay...)

Reply:
Noted.

> [!NOTE]
> Changed the URL to `v3` instead of `gsoc`, and the module name become `typesafe`